src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
Please somebody file an issue proposing this and explaining why it is valuable for Snabb Switch.

If we are going to do it then we should at least be systematic.

I am skeptical. I see this as a practice that projects cargo-cult from each other that is not actually valuable in practice. I am prepared to be convinced though.

@nnikolaev-virtualopensystems I am also a little on guard since you guys at VOSYS do so much branding on the Snabb repository e.g. putting the company name into your handles and putting the company logo as your avatars. I asked you guys to explain why this is important, to see if we could find an alternative way to reach the same goal, and you didn't engage in that discussion. I don't think you guys are being very thoughtful or communicative about this stuff and if other people do it too then it will become a problem.